<launch>
  <arg name="BumblebeeConfigurationFolder" default="/home/ryan/Calibration"/>
  <arg name="publishTransform" default="False"/>
  <node name="feature_node" pkg="front_end" type="feature_node" output="screen"/>
  <node name="bumblebee_configuration" type="bumblebee_configuration.py" output="screen" pkg="bumblebee" args="$(arg BumblebeeConfigurationFolder) bumblebee_configuration $(arg publishTransform)"/>
  <node name="bumblebee" type="bumblebee_node.py" output="screen" pkg="bumblebee" args="bumblebee_configuration"/>
</launch> 